core/domain/suggestion_services.py

```diff
@@ -1187,8 +1196,8 @@ def get_translation_suggestions_in_review_by_exp_ids(
 
 
 def get_suggestions_with_translatable_explorations(
-    suggestions: List[suggestion_registry.SuggestionTranslateContent]
-) -> List[suggestion_registry.SuggestionTranslateContent]:
+    suggestions: Sequence[suggestion_registry.BaseSuggestion]
```

Comment:
With regards to the context, this function only relates to "translate" suggestions, right? If so, then the callers should be clamping down on the type at the point where we first know it's a "translate" suggestion, and this function should use the more specific type rather than BaseSuggestion.

Reply:
reverted back to `suggestion_registry.SuggestionTranslateContent` type.